packages/fast-components-react-msft/src/subheading/subheading.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json.schema.org/schema#",
+    "title": "Subheading",
+    "description": "A Subheading component's schema definition",
+    "type": "object",
+    "id": "Subheading",
+    "children": true,
+    "properties": {
+        "tag": {
+            "title": "HTML Tag",
+            "type": "string",
+            "enum": [
+                "h1",
+                "h2",
+                "h3",
+                "h4",
+                "h5",
+                "h6",
+                "p"                
+            ]
+        },
+        "level": {
+            "title": "Level",
+            "type": "number",
+            "enum": [
+                1,
+                2,
+                3,
+                4,
+                5,
+                6
+            ]

```

Comment:
all of these have defaults of 1 or "h1" in the case of the tag, I would add that in since although they are optional in the component itself they default to a value

Reply:
I think done